Grammar/python.gram

```diff
@@ -1254,8 +1254,10 @@ invalid_try_stmt:
     | a='try' ':' NEWLINE !INDENT {
         RAISE_INDENTATION_ERROR("expected an indented block after 'try' statement on line %d", a->lineno) }
     | 'try' ':' block !('except' | 'finally') { RAISE_SYNTAX_ERROR("expected 'except' or 'finally' block") }
-    | a='try' ':' block* ((except_block+ except_star_block) | (except_star_block+ except_block)) block* {
-        RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "cannot have both 'except' and 'except*' on the same 'try'") }
+    | 'try' ':' block* except_block+ a='except' b='*' expression ['as' NAME] ':' block* {
+        RAISE_SYNTAX_ERROR_KNOWN_RANGE(a, b, "cannot have 'except*' in a 'try' statement with 'except'") }
```

Comment:
I thought this was okay since from a purely syntax perspective, the first one to deviate is the wrong one. But I guess this can make the message a bit harder to understand. Would you proposal to go back to the previous wording?

Reply:
I think so. It seems to make sense that you get the same error msg in both cases.